samples/AzCopyCore/AzCopyCore/AzCopyCore.csproj

```diff
@@ -19,18 +21,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26223-02" />
-    <PackageReference Include="System.Memory" Version="$(CoreFxPackageVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(CoreFxPackageVersion)" />
+    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26320-02" />
+    <PackageReference Include="System.Buffers.ReaderWriter" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Memory" Version="4.5.0-preview3-26319-04" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.0-preview3-26319-04" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Azure.Experimental" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Buffers.Experimental" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Text.Http" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Text.Http.Parser" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Text.Utf8String" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.IO.Pipelines" Version="$(CoreFxLabPackageVersion)" />
+    <PackageReference Include="System.Azure.Experimental" Version="0.1.0-preview2-180320-2" />
```

Comment:
Update and use the CoreFxLabPackageVersion to make it easier to update in the future.

Reply:
Yeah, but there is a trade off here. I think it's good that we have some samples that validate how our customers are using and demonstrate how  they should be using our packages.

Reply:
I don't understand the trade off here. We have all these package version numbers here, and they point to the same version. We have a property in *this* csproj called CoreFxLabPackageVersion. I was just suggesting we use that so that it becomes a single point of edit in the future, if we ever want to upgrade them again. Note, this is still isolated from the rest of the repo and would still demonstrate the customer's use of our packages.